### PR TITLE
Add link to apis of .net link to porting guidelines

### PR DESCRIPTION
**Pull request by zsd4yr**
@OliaG we should probably also add current understood guidance on what to do about multiple app domains, what to do about assembly loader and metadata load context, etc...

### CODE REVIEW

File: Documentation/porting-guidelines.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ differences:
 System.PlatformNotSupportedException: 'Configuration files are not supported.'
 ```
 
+### Additional Types and Namespaces
+
+YOu can search for additional types whcih you may need in porting your apps to .NET Core on [APIs of DotNet][apisofnet]. For example, when you search for System.AppDomain, you will see that it has been moved to System.Runtime.Extensions during the .NET Core 2.0 timeframe is supported on .NET Core 3.0
```
Comment (danmosemsft) [inline Documentation/porting-guidelines.md]: Capitalization of you

Comment (danmosemsft) [reply Documentation/porting-guidelines.md]: "you will see that it has been moved to System.Runtime.Extensions during the .NET Core 2.0 timeframe is supported on .NET Core 3.0" ... I can't parse this

Comment (RussKie) [inline Documentation/porting-guidelines.md]:
```suggestion
You can search for additional types which you may need in porting your apps to .NET Core on [APIs of DotNet][apisofnet]. For example, when you search for `System.AppDomain` type, you will see that it has been moved to `System.Runtime.Extensions` namespace during the .NET Core 2.0 timeframe is supported on .NET Core 3.0
```

As @danmosemsft pointed, the end of the sentence doesn't make any sense.
Did you mean something like the following?
```suggestion
You can search for additional types which you may need in porting your apps to .NET Core on [APIs of DotNet][apisofnet]. For example, when you search for `System.AppDomain` type, you will see that it has been moved to `System.Runtime.Extensions` namespace in .NET Core 3.0.
```

Comment (terrajobst) [reply Documentation/porting-guidelines.md]: Missing “and” before “is supported”

Comment (terrajobst) [reply Documentation/porting-guidelines.md]: Missing “and” before “is supported”

Comment (Shyam-Gupta) [reply Documentation/porting-guidelines.md]: whcih --> which

**Pull request merged**